python/jupytercad_core/jupytercad_core/jcad_ydoc.py

```diff
@@ -42,6 +44,12 @@ def set(self, value: str) -> None:
         :type value: Any
         """
         valueDict = json.loads(value)
+
+        # Assuming file version 3.0.0 if the version is not specified
+        file_version = valueDict["schemaVersion"] if "schemaVersion" in valueDict else "3.0.0"
+        if file_version != CURRENT_SCHEMA_VERSION:
+            raise ValueError(f"Cannot load file version {file_version}")
```

Comment:
I'm raising here, because this is the place we will be reading and adapting the file content to the current schema (e.g. adapting file 3.0.0 to to 4.0.0 in the future).

Thoughts @trungleduc ?

Reply:
A safer bet is to compare the version? We should support any version lower than the current schema version

Reply:
> We should support any version lower than the current schema version

I guess there could be a day when we drop support for files that are too old if we have to. But yes comparing the versions sounds good for now.